### PR TITLE
fix(jest): add LoginManager.logOut + AccessToken mocks

### PR DESCRIPTION
**Pull request by Andarius**
Fix issue where `jest.spyOn(LoginManager, 'logOut')` would raise `Cannot spy the logOut property because it is not a function; undefined given instead`

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]
Nice! Thank you
Don't worry about the conventional commit CI job, that's a quirk of the way GitHub does merges and is just a warning to maintainers at this point to be careful at the merge

**Review by mikehardy — APPROVED** [review]
previously looked good to me, now even better ;-). You want me to hang on while you check through other stuff or you want me to merge as soon as CI goes green?

**Comment by Andarius** [comment]
Hey @mikehardy, I think I covered everything I needed, all my tests are passing now. I made a patch while waiting for a release so up to you :)

**Pull request merged**
